mcs_benchmark_data/pipelines/commonsense_qa/roberta_commonsense_qa_submission_transformer.py

```diff
@@ -30,75 +21,8 @@ def transform(
         **kwds,
     ) -> Generator[_Model, None, None]:
 
-        # Yield submissions
-        # Assumes file name in form "*_[systemname]_submission.jsonl" (e.g. dev_rand_split_roberta_submission.jsonl)
-        yield from self.__transform_submission(submission_data_jsonl_file_path)
-
-        submission_uri = URIRef(f"{self.__URI_BASE}:submission:CommonsenseQA-roberta")
-
-        yield from self.__transform_submission_sample(
-            submission_jsonl_file_path, submission_uri
+        yield from self._transform(

```

Comment:
I don't think this is really reusable code, either, since it's specialized toward jsonl? But we can address that in a different PR.

Reply:
Ok I will check in before I move anything around, because now I am uncertain what code you actually wanted me to put into the parent class.